scripts/packages/anyio_local/.python-version

```diff
@@ -0,0 +1 @@
+3.12
```

Comment:
Annoying, but... `uv build` treats this package as the start of the workspace and it picks up the `.python-version` pin in our repository (3.13) which cannot be satisfied during the build tests (which use 3.12).

The better solution here is to _copy_ this project into the test context instead of using this possibly dirty directory in each test! but I don't want to deal with that here.

Reply:
This is such a mood 😭